digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/service/SyncService.java

```diff
@@ -0,0 +1,7 @@
+package org.digit.health.sync.service;
+
+import org.digit.health.sync.web.models.request.SyncUpDto;
+
+public interface SyncService {
+    String sync(SyncUpDto syncUpDto);
```

Comment:
Create a value object SyncId { string syncId; }
This would be the return type

Reply:
Done